deepchem/models/torch_models/hugging_face_models.py

```diff
@@ -0,0 +1,57 @@
+"""
+Hugging Face Model Wrappers
+"""
+from typing import Optional
+import warnings
+try:
+  from transformers import BertModel, BertConfig
+except ModuleNotFoundError:
+  raise ImportError(
+      "DeepChem's wrappers for HuggingFace transformers require transformers.")
+
+from deepchem.models.torch_models.torch_model import TorchModel
+from deepchem.models.losses import Loss, L2Loss
+
+
+# TODO write new examples
+class BertModelWrapper(TorchModel):
+  """Wrapper for the HuggingFace BertModel, loosely based on dc.attentivefp
+
+  Notes
+  -----
+  This class requires transformers to be installed.
+  """
+
+  def __init__(self,
+               model: BertModel,
```

Comment:
Following wrapping conventions elsewhere (see `GATModel` or `GCNModel`), we shouldn't directly accept a `BertModel`. Instead we should accept the needed arguments and then construct the model internally so the user doesn't have to be exposed to the underlying details